NumPy/Arrays of String and Unicode Values/Basic String Operations in NumPy/task.py

```diff
@@ -2,7 +2,7 @@
 
 
 def read_data(file):
-    text = np.loadtxt(file, delimiter='\n', dtype=np.bytes_)
+    text = np.genfromtxt(file, delimiter='\n', dtype=np.bytes_)
     decoded_text = np.char.decode(text)
     upper_text = np.char.upper(decoded_text)
     return upper_text
```

Comment:
All seems cool, but we lost the blank line at the end of this file, unfortunately